cmd/api/api_server.go

```diff
@@ -22,9 +24,25 @@ var (
 	rdb      *redis.Client
 	ctx      context.Context
 	redisKey = "Stock"
+	logger   *zap.Logger
 )
 
+// ClusterListResponse models the Cluster list response returned by this API
+type ClusterListResponse struct {
+	Count    int                 `json:"count"`
+	Clusters []inventory.Cluster `json:"clusters"`
+}
+
+// AccountListResponse model the Accounts list response returned by this API
+type AccountListResponse struct {
+	Count    int                 `json:"count"`
+	Accounts []inventory.Account `json:"accounts"`
+}
+
 func init() {
+	// Logging config
+	logger, _ = zap.NewProduction()
```

Comment:
Just an observation. **Not a blocker.**

The default config is good enough, but you might want to use a different timestamp.
The current timestamp is from `EpochTime*`, but `ISO8601TimeEncoder` seems more appropriate.

I usually use this config, but it's just to share

```go
encoderCfg := zap.NewProductionEncoderConfig()
encoderCfg.TimeKey = "timestamp"
encoderCfg.EncodeTime = zapcore.ISO8601TimeEncoder

config := zap.Config{
Level: zap.NewAtomicLevelAt(zap.InfoLevel),
development: false
DisableCaller: true
DisableStacktrace: true
Sampling: nil
encoding: "json",
EncoderConfig: encoderCfg,
OutputPaths: []string{
"stdout",
},
ErrorOutputPaths: []string{
"stderr",
},
}

logLevel := strings.ToLower(os.Getenv("LOG_LEVEL"))
if logLevel == "debug" {
config.Level = zap.NewAtomicLevelAt(zap.DebugLevel)
config.DisableStacktrace = false
config.DisableCaller = false
}
logger = zap.Must(config.Build())
```

Reply:
Do you think makes sense to abstract this changes in our own logger module and share with rests of components to include the same config in all of them?
If yes, instead of include this changes harcoded on the API, we could simply import our Logger module with all of those parameters configured by default

Reply:
Yes, the original idea was the same.
I just didn't want to mix it all up.

Reply:
Then, just to don't block this merge, I think we could create an issue for this, and include these changes in a separate PR. Also to don't mix different developments on same PR.

Reply:
Feature request registered here: https://github.com/RHEcosystemAppEng/cluster-iq/issues/25